src/Tasks/Microsoft.NET.Build.Tasks/build/platforms/Portable.v4.5.targets

```diff
@@ -0,0 +1,47 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+
+  <PropertyGroup Condition="$(_PortableTfmCount) == 2 and                             
```

Comment:
I generated them by hand, manually. Without the ability to call a task in static property evaluation, I don't know how to handle optional TFM's like Xamarin. If Xamarin or an invalid one is specified, it won't match and it'll generate an Invalid TFM error.